### PR TITLE
unit leave formation if it is too far away / acceleration fix

### PR DESCRIPTION
**Pull request by Slava0135**
Peek 2020-12-24 16-06
Peek 2020-12-24 16-00

### CODE REVIEW

**Comment by Slava0135** [comment]
I guess the accel = 1.0 was made on purpose. But it felt weird. Also this way you could accelerate yourself before.

File: core/src/mindustry/ai/types/FormationAI.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ public void init(){
     @Override
     public void updateUnit(){
 
-        if(leader == null || leader.dead){
+        if(leader == null || leader.dead || !unit.within(leader.x, leader.y, 10*formation.pattern.spacing)){
```
Comment (Quezler) [inline core/src/mindustry/ai/types/FormationAI.java:30]:
```suggestion
        if(leader == null || leader.dead || !unit.within(leader.x, leader.y, 10 * formation.pattern.spacing)){
```

**Comment by Anuken** [comment]
> But it felt weird. Also this way you could accelerate yourself before.

Why would you want your commanded units to have movement delay? This only breaks things.

**Comment by Slava0135** [comment]
ok

**Pull request closed without merging**